script/web_discovery_project.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+from lib.config import PLATFORM, SOURCE_ROOT, enable_verbose_mode
+from lib.util import execute_stdout
+
+WEB_DISCOVERY_DIR = os.path.join(SOURCE_ROOT, 'vendor', 'web-discovery-project')
+
+NPM = 'npm'
+if PLATFORM in ['win32', 'cygwin']:
+    NPM += '.cmd'
+
+
+def main():
+    os.chdir(WEB_DISCOVERY_DIR)
+    args = parse_args()
+
+    if args.verbose:
+        enable_verbose_mode()
```

Comment:
I don't think this does anything by itself

Reply:
it does. it works with the stdout commands inside the same lib.

Reply:
other usages (like bootstrap.py) do this
```        if is_verbose_mode():
            args += ['--verbose']```

Reply:
but I guess I did miss 
```
def execute_stdout(argv, env=os.environ):
    if is_verbose_mode():
```
when I looked before